### PR TITLE
Journal list page layout change and add latest issue box in journal page

### PR DESCRIPTION
**Pull request by charanjeetcs**

### CODE REVIEW

File: web/themes/highwire/hwjma_theme/apps/drupal-default/hwjma_theme/includes/node.theme.inc [hunk 1]
```diff
@@ -236,6 +236,70 @@ function hwjma_preprocess_node__journal__content_details(&$variables) {
...
     }
+
+   //JCOREX-96:Latest issue details add Journal page 
+    $journal_current_issue = $node->get('journal_current_issue')->getValue()[0]['target_id'];
+    if (!empty($journal_current_issue)) {
+    $latest_issue_node = Node::load($journal_current_issue);
+    $volume_no = $latest_issue_node->get('volume')->getString();
+    $issue_no = $latest_issue_node->get('issue')->getString();
+    $imagefield = $latest_issue_node->get('variant_cover_image')->getValue();
+    $media_url = '/themes/highwire/hwjma_theme/dist/app-drupal/assets/images/250X350.png';
```
Comment (RahulSatija49) [inline web/themes/highwire/hwjma_theme/apps/drupal-default/hwjma_theme/includes/node.theme.inc:247]: Can see image path static.  can we do this dynamic or can do better.

**Review by RahulSatija49 — COMMENTED** [review]
Check one comment

File: web/themes/highwire/hwjma_theme/apps/drupal-default/hwjma_theme/includes/node.theme.inc (hunk 1 above)
Comment (charanjeetcs) [reply web/themes/highwire/hwjma_theme/apps/drupal-default/hwjma_theme/includes/node.theme.inc]: Remove Img path

**Comment by RahulSatija49** [comment]
Code is looking good, passing for QA

**Pull request merged**